src/plugins/teleport-plugin-vue-base-component/utils.ts

```diff
@@ -33,7 +34,7 @@ export const generateVueNodesTree = (
 
   if (events) {
     Object.keys(events).forEach((eventKey) => {
-      const methodName = `handle${capitalize(name)}${capitalize(eventKey)}`
+      const methodName = `handle${stringToCamelCase(name)}${stringToCamelCase(eventKey)}`
```

Comment:
does this output: `handlebuttonclick` or `handleButtonClick`? the latter should be the case and I think you need to create another function that does `capitalize(stringToCamelCase(string))`. you can actually extract it in `string-utils` and name it `stringToUpperCamelCase`